### PR TITLE
Ctrl+N/P/J/K navigation (#146)

### PR DESCRIPTION
**Pull request by jminjie**
Navigate down and up using Ctrl+N/P or Ctrl+J/K

### CODE REVIEW

File: app/renderer-jsx/app.jsx [hunk 1]
```diff
@@ -268,10 +268,31 @@ class AppContainer extends React.Component {
...
     };
+    const ctrlKeyHandlers = {
+      P: this.handleSelection.bind(this, -1),
+      p: this.handleSelection.bind(this, -1),
+      K: this.handleSelection.bind(this, -1),
+      k: this.handleSelection.bind(this, -1),
+      N: this.handleSelection.bind(this, 1),
+      n: this.handleSelection.bind(this, 1),
+      J: this.handleSelection.bind(this, 1),
+      j: this.handleSelection.bind(this, 1)
```
Comment (blainesch) [inline app/renderer-jsx/app.jsx]: Do we need the capped versions of these?

Comment (jminjie) [inline app/renderer-jsx/app.jsx]: No they aren't necessary, good point

**Comment by appetizermonster** [comment]
Actually, I don't know why we should add this shortcuts. Is it common shortcuts?

**Comment by blainesch** [comment]
@appetizermonster it's related to #146 and yes, it's popular for most extensions to have the ctrl+n/ctrl+p to scroll up/down the list. This is present in apps like chrome, terminal, slack and other popular apps.

File: app/renderer-jsx/app.jsx (hunk 1 above)
Comment (jminjie) [inline app/renderer-jsx/app.jsx]: Actually these shortcuts generally work even if Caps Lock is on. It wouldn't be a bad idea to include the capitalized letters.

**Pull request merged**